modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Tyranid armies:
+
+* The army does not use objectives, regardless of scenario.
```

Comment:
A couple of choices of bullet point to choose from, depending on the final choice for army morale phrasing...